napalm_junos/junos.py

```diff
@@ -51,6 +51,10 @@ def open(self):
         except ConnectTimeoutError as cte:
             raise ConnectionException(cte.message)
         self.device.timeout = self.timeout
+        if hasattr(self.device, "cu"):
```

Comment:
So all you can do is to re-open the connection. But if the Config is already bind cannot properly execute ```open```, hence my changes.